upgrade_tests/helpers/variants.py

```diff
@@ -544,6 +585,19 @@ class VersionError(Exception):
         '+         cmd: |',
         "- <%=     indent(8) { input('command') } %>",
         "+ <%=       indent(10) { input('command') } %>",
+        # Display all parameters known for the Foreman host
```

Comment:
We need a template name, this more looks like a description :)

Reply:
Fixed